compiler/src/org.graalvm.compiler.core.common/src/org/graalvm/compiler/core/common/type/FloatStamp.java

```diff
@@ -910,6 +910,76 @@ public Stamp foldStamp(Stamp s) {
 
                     null, null, null,
 
+                    new BinaryOp.Max(true, true) {
```

Comment:
Since Max/Min are associative, I believe also hooks for them must be added into `BinaryArithmeticNode.reassociate`

Reply:
Done!